ui/admin/app/templates/scopes/scope/targets/new.hbs

```diff
@@ -17,7 +17,7 @@
   <page.body>
     <Form::Target
       @model={{@model}}
-      @submit={{route-action 'save' @model}}
+      @submit={{route-action 'saveWithAddress' @model}}
```

Comment:
A new target cannot have host sources, since it must first be saved before they can be added.
```suggestion
      @submit={{route-action 'save' @model}}
```